packages/client/src/client/App/Credit/CreditRfqForm/CreditRfqFormCore.tsx

```diff
@@ -65,7 +67,8 @@ const CreditRfqFormCore = ({ children }: WithChildren) => {
   }, [])
   return (
     <CreditRfqFormCoreWrapper>
-      <Subscribe fallback={children}>
+      <Subscribe fallback={<Loader ariaLabel="Loading New RFQ Form" />}>
```

Comment:
why has this changed?
I had made this consistent with other *`Core` impls by using one version of Loader in parent and in the Core inner component ..

Reply:
I haven't changed this, maybe due to merging with master, I accepted the wrong changes will revert.